README.md

```diff
@@ -7,6 +7,35 @@ jitsi-meet-electron-utils contains native code for some utilities. You'll need [
 NOTE: For Linux install libxtst-dev and libpng++-dev (`sudo apt-get install libxtst-dev libpng++-dev`). This dependancies are related to RobotJS which is a dependency of jitsi-meet-electron-utils. You can see the build instructions for RobotJS [here](https://github.com/jitsi/robotjs/tree/jitsi#building)
 
 ## Usage
+#### Google API
+The Google API utility exposes methods for performing Google client-side authentication by opening a oauth popup window and passing back the redirect URL with the Google access token.
+
+**Enable the Google API integration:**
+
+In the **main** electron process:
+```Javascript
+const { setupGoogleApiMain } = require("jitsi-meet-electron-utils");
+const options = {
+    browserWindowOptions: {
+        webPreferences: {
+            nodeIntegration: false
+        }
+    }
+};
+setupGoogleApiMain(options);
+```
+
+In the **render** electron process of the window where Jitsi Meet is displayed:
+```Javascript
+const { setupGoogleApiRender } = require("jitsi-meet-electron-utils");
+const api = new JitsiMeetExternalAPI(...);
+setupGoogleApiRender(api);
+```
+
+To clean up Google API integration:
+```
+teardownGoogleApi();
```

Comment:
This line should be called in the main process or the render process or it doesn't matter?

Reply:
Hmm...Yeah, should be called in the main process because it removes listeners that are applied to ipc main. I'll specify that